test/libsolidity/semanticTests/fallback/call_forward_bytes.sol

```diff
@@ -0,0 +1,26 @@
+contract receiver {
+    uint256 public received;
+    function recv(uint256 x) public { received += x + 1; }
+    fallback() external { received = 0x80; }
+}
+contract sender {
+    constructor() { rec = new receiver(); }
+    fallback() external { savedData = msg.data; }
+    function forward() public returns (bool) { address(rec).call(savedData); return true; }
+    function clear() public returns (bool) { delete savedData; return true; }
+    function val() public returns (uint) { return rec.received(); }
+    receiver rec;
+    bytes savedData;
+}
+// ====
+// compileToEwasm : false
+// compileViaYul: also
+// ----
+// (): 7 ->
```

Comment:
```suggestion
// recv(uint256): 7 ->
```

Reply:
We get this error, when we use recv(uint256) instead of () . 
`semanticTests/fallback/call_forward_bytes.sol: Exception during test: /solidity/test/libsolidity/SemanticTest.cpp(436): Throw in function solidity::frontend::test::TestCase::TestResult solidity::frontend::test::SemanticTest::runTest(std::ostream&, const string&, bool, bool, bool)
Dynamic exception type: boost::wrapexcept<std::runtime_error>
std::exception::what: The function recv(uint256) is not known to the compiler
`

We figured that, calling recv() should only invoke the fallback function of the sender contract. So, we used a constructor to make that happen

Reply:
I've added a call to rec.recv(7) inside the constructor in the latest push.
Will that be okay ?

Reply:
You can prevent the error by adding `// allowNonExistingFunctions: true` to the settings of the test. Please don't change the test from how it was.

Reply:
Reverted the change. Added the  `//allowNonExistingFunctions: true` line to the settings of the test